developer/kmcompx/tests/get_test_source_exclude_some_kmn.bat

```diff
@@ -0,0 +1,21 @@
+@echo off
+
+rem excluded files from tests:
+
+rem no *bmp file included in Folder
+rem		- eKwTamil99UniUpdt
+
+rem use of other folders than /source
+rem		- vn_telex/raw/header
```

Comment:
where are these excluded? I don't see them in the `dir` line below.

Reply:
they are excluded using viet* from
keyboards\release\v\vietnamese_telex_legacy\extras\vn_kmn_compiler\vn_telex\raw
keyboards\release\v\vietnamese_vni\extras\vn_kmn_compiler\keyboard\vn_vni\raw